src/backend/distributed/citus--7.0-1.sql

```diff
@@ -0,0 +1,1735 @@
+/* citus.sql */
+
+-- complain if script is sourced in psql, rather than via CREATE EXTENSION
+\echo Use "CREATE EXTENSION citus" to load this file. \quit
+
+CREATE SCHEMA citus;
+
+-- Ensure CREATE EXTENSION is not run against an old citus data
+-- directory, we're not compatible (due to the builtin functions/tables)
+DO $$
+BEGIN
+   IF EXISTS(SELECT * FROM pg_proc WHERE proname = 'worker_apply_shard_ddl_command') THEN
+      RAISE 'cannot install citus extension in Citus 4 data directory';
```

Comment:
Update the check and the error message.

Reply:
Removed altogether since postgres versions shouldn't match up anyways